components/feature/push/src/main/java/mozilla/components/feature/push/AutoPushFeature.kt

```diff
@@ -371,6 +374,9 @@ internal inline fun exceptionHandler(crossinline onError: (PushError) -> Unit) =
         is GeneralError,
         is CryptoError,
         is CommunicationError,
+        is JSONDeserializeError,
+        is RequestError,
+        is UAIDNotRecognizedError,
```

Comment:
Thinking about it again, if we mark this as fatal we can then monitor the numbers on Sentry to see if it reduces with the bug fixes from a-s in the future.

Reply:
Sounds good, I've changed it.